wordpress-mobile/MediaPicker-iOS
# Implement delegate method to allow library users to handle errors.

Has the title say this PR just brings a new API for the delegate class to allow users of the library to customize the way errors are displayed and handled to users.

This will allow us to better handle issues like this one: https://github.com/wordpress-mobile/WordPress-iOS/issues/11320 , where the main app is already handling the error on their UI and we display on top.

To test:
 - Make sure that all builds correctly.
 - Run the demo app.
